server/routes/events.js

```diff
@@ -12,101 +13,507 @@ router.get('/ping', function(req, res, next) {
 
 // GET all events
 router.get('/', function(req, res, next) {
-  db.query('SELECT * FROM event', [], (err, result) => {
-    if (err) return next(err);
-    res.send(result.rows);
-  });
+  knex('event').select()
+  .then(result => {
+    if (result.length) {
+      res.json(result);
+    } else {
+      res.status(404).json('No events found.');
+    }
+  })
+  .catch(err => { return next(err) });
+});
+
+// Add a single event
+router.post('/', function(req, res, next) {
+	let errormsg = 'The following fields must not be null: ';
```

Comment:
Same thing as I said before, just do a basic error message rather than concatenating a string/complicating it